ui/src/lib/gql/queries/user.ts

```diff
@@ -16,13 +15,14 @@ graphql(`
   }
 `);
 
+export enum UserQueryKeys {
+  GetMe = "getMe",
+}
+
 export const useMeQuery = () => {
   const graphQLContext = useGraphQLContext();
-  const { data, ...rest } = useQuery({
-    // TODO: Use static keys rather than strings. Export the keys as well
-    queryKey: ["getMe"],
-    queryFn: async () => graphQLContext?.GetMe(),
+  return useQuery({
+    queryKey: [UserQueryKeys.GetMe],
+    queryFn: () => graphQLContext?.GetMe(),
```

Comment:
I thought the `queryFn` needs to be async 🤔

Reply:
Not necessarily, the Tanstack automatically figures it out.

Reply:
I could be wrong, but I think if we wrap an async func in a normal func where we use that func will not await the child. Which we want.
Also, [the docs](https://tanstack.com/query/v4/docs/framework/react/guides/query-functions) show making the queryFn async so should stick with the recommended way

Reply:
The logic is you can return a Promise and it'll await that. If it's just the fn we don't need to write `async`, in the doc second example it's not using `async`